.zuul.yaml

```diff
@@ -1,16 +1,23 @@
 ---
+- job:
+    name: mypy-with-dependencies
```

Comment:
The job name should start with the repo name, since the namespace for jobs spans the whole tenant with all repos in it. If you want to re-use the job for other repos, add it in the zuul-jobs repo instead.